psqlgraph/node.py

```diff
@@ -142,6 +142,26 @@ def copy(self):
         )
         return node
 
+    def to_json(self):
+        return {
+            'node_id': self.node_id,
+            'label': self.label,
+            'acl': self.acl,
+            'properties': self.properties,
+            'system_annotations': self.system_annotations,
+            'edges_out': [e.to_json() for e in self.edges_out],
+            'edges_in': [e.to_json() for e in self.edges_in]
+        }
+
+    @classmethod
+    def from_json(cls, node_json):
+        Type = Node.get_subclass(node_json['label'])
+        return Type(node_id=node_json['node_id'],
```

Comment:
Not sure why I can't initialize by using cls(), but for some reason, that seems to run __init__ on the super class  setting the label to the default of the class name and thereby not not allowing it to be set again.